### PR TITLE
[EXPLORER] Improve Portuguese (pt-PT) translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
Improve and fix typo to pt-PT rc file
Fix SUBLANG type to pt-PT language

### CODE REVIEW

File: base/shell/explorer/lang/pt-PT.rc [hunk 1]
```diff
@@ -156,14 +161,14 @@ BEGIN
     LTEXT            "Items do &menu 'Iniciar':", IDC_STATIC, 7, 53, 238, 8
     CONTROL          "StartMenuSe&ttings", IDC_ITEMS, "SysTreeView32", WS_BORDER | 0x00000001, 7, 63, 238, 67, WS_EX_CLIENTEDGE
     GROUPBOX         "Documentos recentes", IDC_STATIC, 7, 133, 238, 56
-    LTEXT            "Seleccione esta opção para ter rápido acesso aos documentos abertos recentemente.  Limpando esta lista, não apaga os documentos.", IDC_STATIC, 14, 144, 224, 21
+    LTEXT            "Seleccione esta opção para ter rápido acesso aos documentos abertos recentemente.  Ao limpar esta lista, não apaga os documentos.", IDC_STATIC, 14, 144, 224, 21
```
Comment (JoachimHenze) [inline base/shell/explorer/lang/pt-PT.rc:164]:
```suggestion
    LTEXT            "Seleccione esta opção para ter rápido acesso aos documentos abertos recentemente. Ao limpar esta lista, não apaga os documentos.", IDC_STATIC, 14, 144, 224, 21
```

File: base/shell/explorer/lang/pt-PT.rc [hunk 2]
```diff
@@ -172,7 +177,7 @@ BEGIN
     AUTORADIOBUTTON  "Icons &grandes", IDC_CHOOSELARGE, 43, 25, 53, 10
     AUTORADIOBUTTON  "icones &pequenos", IDC_CHOOSESMALL, 177, 25, 51, 10, NOT WS_TABSTOP
     GROUPBOX         "Programas ", IDC_STATIC, 7, 49, 238, 70
-    LTEXT            "O menu 'Iniciar' comtém atalhos para os programas que usa frequentemente.  limpando a lista de atalhos, não elimina os programas.", IDC_STATIC, 14, 63, 224, 20
+    LTEXT            "O menu 'Iniciar' comtém atalhos para os programas que usa frequentemente.  Ao limpar a lista de atalhos, não elimina os programas.", IDC_STATIC, 14, 63, 224, 20
```
Comment (JoachimHenze) [inline base/shell/explorer/lang/pt-PT.rc:180]:
```suggestion
    LTEXT            "O menu 'Iniciar' comtém atalhos para os programas que usa frequentemente. Ao limpar a lista de atalhos, não elimina os programas.", IDC_STATIC, 14, 63, 224, 20
```

File: base/shell/explorer/lang/sk-SK.rc [hunk 3]
```diff
@@ -1,5 +1,8 @@
-/* TRANSLATOR : Mário Kačmár /Mario Kacmar/ aka Kario (kario@szm.sk)
- * DATE OF TR.: 20-07-2007
+/*
+ * PROJECT:     Explorer for ReactOS
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Slovak Language resource file
+ * TRANSLATORS: Mário Kačmár /Mario Kacmar/ aka Kario (kario@szm.sk) DATE OF TR.: 20-07-2007
  * LAST UPDATE: 26-07-2010
```
Comment (JoachimHenze) [inline base/shell/explorer/lang/sk-SK.rc:6]:
```suggestion
```

File: base/shell/explorer/lang/sq-AL.rc [hunk 4]
```diff
@@ -1,5 +1,8 @@
-/* TRANSLATOR : Ardit Dani (Ard1t) (ardit.dani@gmail.com)
- * DATE OF TR:  29-11-2013
+/*
+ * PROJECT:     Explorer for ReactOS
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Albanian Language resource file
+ * TRANSLATORS: Ardit Dani (Ard1t) (ardit.dani@gmail.com) DATE OF TR:  29-11-2013
```
Comment (JoachimHenze) [inline base/shell/explorer/lang/sq-AL.rc:5]:
```suggestion
 * TRANSLATORS: 2013 Ardit Dani (Ard1t) (ardit.dani@gmail.com)
```

File: base/shell/explorer/lang/sk-SK.rc [hunk 5]
```diff
@@ -1,5 +1,8 @@
-/* TRANSLATOR : Mário Kačmár /Mario Kacmar/ aka Kario (kario@szm.sk)
- * DATE OF TR.: 20-07-2007
+/*
+ * PROJECT:     Explorer for ReactOS
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Slovak Language resource file
+ * TRANSLATORS: Mário Kačmár /Mario Kacmar/ aka Kario (kario@szm.sk) DATE OF TR.: 20-07-2007
```
Comment (JoachimHenze) [inline base/shell/explorer/lang/sk-SK.rc:5]:
```suggestion
 * TRANSLATORS: 2007 Mário Kačmár /Mario Kacmar/ aka Kario (kario@szm.sk)
```

File: base/shell/explorer/lang/pt-PT.rc [hunk 6]
```diff
@@ -1,6 +1,11 @@
-/* Translation and UTF-8 Conversion by mkbu95 <mkbu95@gmail.com> (May, 2012) */
+/*
+ * PROJECT:     Explorer for ReactOS
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Portuguese Language resource file
+ * TRANSLATORS: mkbu95 <mkbu95@gmail.com> (May, 2012)
```
Comment (JoachimHenze) [inline base/shell/explorer/lang/pt-PT.rc:5]:
```suggestion
 * TRANSLATORS: 2012 mkbu95 <mkbu95@gmail.com>
```

File: base/shell/explorer/lang/pt-BR.rc [hunk 7]
```diff
@@ -1,4 +1,9 @@
-/* Translation and UTF-8 Conversion by mkbu95 <mkbu95@gmail.com> (May, 2012) */
+/*
+ * PROJECT:     Explorer for ReactOS
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Portuguese Brazilian Language resource file
+ * TRANSLATORS: and UTF-8 Conversion by mkbu95 <mkbu95@gmail.com> (May, 2012)
```
Comment (JoachimHenze) [inline base/shell/explorer/lang/pt-BR.rc:5]:
```suggestion
 * TRANSLATORS: 2012 mkbu95 <mkbu95@gmail.com>
```

File: base/shell/explorer/lang/pl-PL.rc [hunk 8]
```diff
@@ -1,8 +1,8 @@
 /*
- *    Translated by Caemyr - Olaf Siejka (Jun, 2007)
- *    Use ReactOS forum PM or IRC to contact me
- *    https://reactos.org/
- *    IRC: irc.freenode.net #reactos-pl;
+ * PROJECT:     Explorer for ReactOS
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Polish Language resource file
+ * TRANSLATORS: Caemyr - Olaf Siejka (Jun, 2007) to contact me https://reactos.org/ IRC: irc.freenode.net #reactos-pl;
```
Comment (JoachimHenze) [inline base/shell/explorer/lang/pl-PL.rc:5]:
```suggestion
 * TRANSLATORS: 2007 Caemyr - Olaf Siejka to contact me https://reactos.org/ IRC: irc.freenode.net #reactos-pl;
```

File: base/shell/explorer/lang/ms-MY.rc [hunk 9]
```diff
@@ -1,4 +1,9 @@
-/* Malay translation by Henry Tang Ih 2016 (henrytang2@hotmail.com) */
+/*
+ * PROJECT:     Explorer for ReactOS
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Malay Language resource file
+ * TRANSLATORS: Henry Tang Ih 2016 (henrytang2@hotmail.com)
```
Comment (JoachimHenze) [inline base/shell/explorer/lang/ms-MY.rc:5]:
```suggestion
 * TRANSLATORS: 2016 Henry Tang Ih (henrytang2@hotmail.com)
```

**Comment by HBelusca** [comment]
Adding Stas for checking the file headers :)

File: base/shell/explorer/lang/bg-BG.rc [hunk 10]
```diff
@@ -1,3 +1,10 @@
+/*
+ * PROJECT:     Explorer for ReactOS
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Bulgarian Language resource file
+ * TRANSLATORS:
+ */
+
```
Comment (binarymaster) [inline base/shell/explorer/lang/bg-BG.rc:7]: Re. headers, I would rather do this myself in a separate commit, this kind of maintenance should be done by the core team.

Comment (binarymaster) [reply base/shell/explorer/lang/bg-BG.rc]: UPD: committed update for headers in master and resolved the conflict here now.

File: base/shell/explorer/lang/pt-PT.rc [hunk 11]
```diff
@@ -7,7 +7,7 @@
  *              Copyright 2021 Gabriel Aguiar <fgsoftwarestudio@gmail.com>
  */
 
-LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
```
Comment (learn-more) [inline base/shell/explorer/lang/pt-PT.rc:10]: The majority of files has `SUBLANG_NEUTRAL`, is this something that recently changed?

Comment (zecarlos1957) [reply base/shell/explorer/lang/pt-PT.rc:10]: I have mistakenly defined SUBLANG_NEUTRAL. After reading more carefully the text at https://reactos.org/wiki/Sublang, I see that SUBLANG_PORTUGUESE is the correct one when there is a translation for SUBLANG_PORTUGUESE_BRAZILIEN, and SUBLANG_NEUTRAL when there is no translation for Brazilian.

File: base/shell/explorer/lang/pt-PT.rc [hunk 12]
```diff
@@ -204,6 +204,6 @@ STRINGTABLE
 BEGIN
     IDS_TASKBAR_STARTMENU_PROP_CAPTION "Barra de Tarefas e Menu Iniciar"
     IDS_RESTORE_ALL "&Mostrar janelas abertas"
-    IDS_TRAYWND_UNDO_CASCADE "&Undo Cascade"
-    IDS_TRAYWND_UNDO_TILE "&Undo Tile"
+    IDS_TRAYWND_UNDO_CASCADE "Desfazer cascata"
+    IDS_TRAYWND_UNDO_TILE "Desfazer lado a lado"
```
Comment (binarymaster) [inline base/shell/explorer/lang/pt-PT.rc:208]: Mnemonic keys (&) missing.

Comment (zecarlos1957) [reply base/shell/explorer/lang/pt-PT.rc]: Thanks

**Review by binarymaster — APPROVED** [review]
